src/service-worker.js

```diff
@@ -47,3 +47,24 @@ workbox.routing.registerRoute(
     plugins: [new workbox.expiration.Plugin({ maxEntries: 10 })],
   }),
 );
+
+self.addEventListener("activate", () => {
+  self.clients.claim();
+});
+
+self.addEventListener("fetch", (event) => {
```

Comment:
I’m not entirely sure this is needed. I believe Workbox does this automatically, but not sure about the configuration in create-react-app. Please verify.

Reply:
Are you referring to `  self.clients.claim();`?

Reply:
Yes.

Reply:
https://github.com/excalidraw/excalidraw/blob/00ac804dd867d214c455292397206233fc784157/src/service-worker.js#L32

True, it's not needed!

Reply:
Perfect. Thanks for checking. I was only 75% sure.